server/src/main/java/org/elasticsearch/search/aggregations/AggregatorFactories.java

```diff
@@ -311,8 +315,10 @@ public AggregatorFactories build(QueryShardContext queryShardContext, Aggregator
             if (skipResolveOrder) {
```

Comment:
While we're here, is it possible to nuke `skipResolveOrder` too?  I believe it's _only_ used by `BasePipelineAggregationTestCase`, and that doesn't even invoke `build()` so this is basically "dead" testing code.  I think.

Not a problem to leave if there's a complication... I just particularly dislike this little tidbit and wouldn't mind seeing it go if we're already touching this :)